config/canonical.json

```diff
@@ -4268,6 +4276,7 @@
   },
   "amenity/bank|Ощадбанк": {
     "count": 958,
+    "countryCodes": ["ua"],
     "tags": {
       "amenity": "bank",
       "brand": "Ощадбанк",
```

Comment:
Are the following wiki entries not correct?
https://www.wikidata.org/wiki/Q4340839
https://uk.wikipedia.org/wiki/%D0%9E%D1%89%D0%B0%D0%B4%D0%B1%D0%B0%D0%BD%D0%BA

Reply:
I had not found this one, but they seem correct, indeed.

Reply:
OK, just push a commit with the changes and I'll approve the PR.

Reply:
That should be done in commit 8316ffb ? Thanks for the help !